addons/repair/models/repair.py

```diff
@@ -706,7 +708,9 @@ class RepairLine(models.Model):
         copy=False, readonly=True, required=True,
         help='The status of a repair line is set automatically to the one of the linked repair order.')
     tracking = fields.Selection(string='Product Tracking', related="product_id.tracking")
-
+    tax_calculation_rounding_method = fields.Selection(
+        related='company_id.tax_calculation_rounding_method',
+        string='Tax calculation rounding method', readonly=True)
```

Comment:
https://peps.python.org/pep-0008/#blank-lines

> Method definitions inside a class are surrounded by a single blank line.

Reply:
Field doesn't seem necessary because there is no kanban. Which seems to be an error for mobile... (didn't test)

Reply:
By removing the field i got an error about field does not exist in model repair.order, so it don't seem to work ?

Reply:
uh?